composer.json

```diff
@@ -29,7 +29,7 @@
         "ext-json": "*",
         "ext-dom": "*",
         "ext-simplexml": "*",
-        "phpunit/phpunit": "~4.0|~5.0",
+        "phpunit/phpunit": "~4.8",
```

Comment:
I'll add this task to backLog to replace PHPUnit_Framework_TestCase with \PHPUnit\Framework\TestCase.
Have updated the composer.json file with values: ^4.8.35|^5.4.0",